kyo-direct/shared/src/test/scala/kyo/CoreTest.scala

```diff
@@ -0,0 +1,195 @@
+package kyo
```

Comment:
I've removed `DirectTest` and replaced it with more complete `PreludeTest` and `CoreTest` suites